.circleci/config.yml

```diff
@@ -170,36 +170,20 @@ jobs:
       - *make_out_dirs
       - attach_workspace:
           at: /tmp/circleci-workspace
-      - run:
-          name: Cloning
-          command: |
-            git clone https://github.com/ipfs/interop.git
-            git -C interop log -1
-      - restore_cache:
-          keys:
-            - v2-interop-{{ checksum "~/ipfs/go-ipfs/interop/package-lock.json" }}
-            - v2-interop-
       - run:
           name: Installing dependencies
           command: |
-            npm install
-          working_directory: ~/ipfs/go-ipfs/interop
-      - save_cache:
-          key: v2-interop-{{ checksum "~/ipfs/go-ipfs/interop/package-lock.json" }}
-          paths:
-            - ~/ipfs/go-ipfs/interop/node_modules
-      - run:
-          name: Installing reporting tools
-          command: |
-            npm install --save-dev mocha-circleci-reporter@0.0.3
+            npm init -y
+            npm install ipfs@^0.48.1
+            npm install ipfs-interop@^2.0.0
+            npm install mocha-circleci-reporter@0.0.3
           working_directory: ~/ipfs/go-ipfs/interop
       - run:
           name: Running tests
           command: |
             mkdir -p /tmp/test-results/interop/
             export MOCHA_FILE="$(mktemp /tmp/test-results/interop/unit.XXXXXX.xml)"
-            node_modules/.bin/mocha --reporter mocha-circleci-reporter \
-              $(sed -n -e "s|^require('\(.*\)')$|test/\1|p" test/node.js | circleci tests split)
+            node_modules/.bin/ipfs-interop -- -t node -f $(sed -n -e "s|^require('\(.*\)')$|test/\1|p" node_modules/ipfs-interop/test/node.js | circleci tests split) -- --reporter mocha-circleci-reporter
```

Comment:
The old version ran the mocha dep that is installed as a side-effect of the interop suite depending on aegir.  It uses sed to print out all the node test filenames which are piped to the `circleci` command which selects a few of the test files to run on this worker.

The new version uses the [ipfs-interop](https://github.com/ipfs/interop/blob/master/bin/ipfs-interop.js) command to do the same thing, except it runs the [npm 'test' script](https://docs.npmjs.com/cli/run-script), which sets up the env and uses aegir to run the tests rather than invoking mocha directly.

Apart from that the change is just the updated path to the `node.js` test file - previously the interop suite was cloned and the tests were run from inside the clone, now a temporary project is created and the interop suite is installed as a dependency of that project so the path has changed.